modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/ExtensionProperties.java

```diff
@@ -50,6 +51,11 @@
   public static final String ENCODING_PARAMETER_NAME = "outputEncoding";
 
   /**
+   * The name of a file which contains the {@link ExtensionModel} elements descriptions.
+   */
+  public static final String EXTENSION_DESCRIPTIONS_FILE_NAME = "extension-descriptions.xml";
```

Comment:
no test file? How are you testing that the jdcos are being correctly persisted and correctly loaded back?